src/hooks/useCourtContracts.js

```diff
@@ -442,3 +451,57 @@ export function useTotalActiveBalancePolling(termId) {
 
   return totalActiveBalance
 }
+
+export function useTotalANTStaked() {
```

Comment:
I think this is currently only reporting the **ANJ** staked (`JurorsRegistry` doesn't know anything about ANT).

For mainnet, total **ANT** staked comes from two places: the Network DAO's Agent, and the Network Fundraising's Vault. You can see how [anj.aragon.org](https://github.com/aragon/anj.aragon.org/blob/master/src/web3-contracts.js#L359) reports this figure.

For rinkeby / other environments, I would just leave this value at 0.

Reply:
ohh yes we got confused with this one! thxs!!